db/seeds.rb

```diff
@@ -27,6 +29,22 @@
 
 puts ">>> SEEDING ENVIRONMENT: #{Rails.env}"
 
+if Region.all.empty?
```

Comment:
@weedySeaDragon - you're right.  I misremembered the method.

What I meant to refer to is `exists?`.  http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-exists-3F

It will try to fetch one record from the table, and return `true` if that succeeds and `false` otherwise.  This would tend to be faster than `select count(*)` for a larger table.   For these tables it probably does not provide mucf of a performance improvement.

FYI: Calling `blank?` on a active record model for which a table does not exists will return `true`, and `false` otherwise.  (that is, it returns `true` if the table does not exist).  On the other hand, the `exists?` method has nothing to do with determining if the table actually _exists_.  Go figure ....